Atomic/syscontainers.py

```diff
@@ -1597,9 +1606,12 @@ def uninstall(self, name):
         installed_files = None
         with open(os.path.join(checkout, name,  "info"), 'r') as info_file:
             info = json.loads(info_file.read())
-            installed_files = info["installed-files"] if "installed-files" in info else None
-        if installed_files:
-            RPMHostInstall.rm_add_files_to_host(installed_files, None)
+            installed_files_checksum = info["installed-files-checksum"] if "installed-files-checksum" in info else None
+            installed_files = info["installed-files"] if "installed-files" in info else []
```

Comment:
Minor: seems like the assignment of checksum here is a bit inconsistent with other implementations, 
```
 installed_files_checksum = info["installed-files-checksum"] if "installed-files-checksum" in info else None
        if installed_files_checksum is None:
            installed_files = info["installed-files"] if "installed-files" in info else None
            installed_files_checksum = {k : "" for k in installed_files}

```
any reasons for that?

Reply:
yes, it is done a bit differently so if installed_files_checksum is not present (for containers installed before this patch), we assume it is empty:

`installed_files_checksum = {k: "" for k in installed_files}`